run_docker.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+docker build -t torch-mlir-cmake:dev .
```

Comment:
do you want to rebuild docker everytime you run it ?

Reply:
Yes. Here's why I think this is inexpensive and useful: For the local workflow I expect users would `edit->run_docker->run_test` in a tight loop and because `docker build` uses layered caching (quite aggressively), unless the last image was invalidated (by a change to Dockerfile itself or `docker rmi`) it'd just use the same to spawn a container. For any source code changes, these would be transparent to the docker build step because we're loading the source tree as a virtual mount and not something baked into the image itself. Does that make sense?

Reply:
The release script should provide us the trampoline into docker seamlessly

Reply:
I agree, this step becomes moot if we're using the base docker image as-is (without a Dockerfile).